README.md

```diff
@@ -1,62 +1,74 @@
-# Getting started with CPU Stats on Mbed OS
+![](./resources/official_armmbed_example_badge.png)
+# CPU Statistics Mbed OS Example
 
-This guide reviews the steps required to get CPU statistics on Mbed OS platform.
+This guide reviews the steps required to get CPU statistics and usage on Mbed OS platform.
 
-Please install [mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+The project can be built with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
 
-## Import the example application
+Please install Arm Mbed CLI.
 
-From the command-line, import the example:
+Clone this repository on your system and change the current directory to where the project was cloned.
 
-```
-mbed import mbed-os-example-cpu-stats
-cd mbed-os-example-cpu-stats
-```
+## Application functionality
 
-### Now compile
+- The `main()` function starts a thread that blinks an LED at varying intervals to keep the CPU busy.
+- It also creates an event queue and adds a function to periodically print the below statistics
```

Comment:
I would suggest not to use bullets for the two sentences above.
You should add `:` after statistics.